sample-211/src/main/java/com/infullmobile/android/infullmvp/sample/dagger11/sample_mvp_activity/di/SampleMvpActivityModule.java

```diff
@@ -0,0 +1,40 @@
+package com.infullmobile.android.infullmvp.sample.dagger11.sample_mvp_activity.di;
+
+import android.content.Context;
+
+import com.infullmobile.android.infullmvp.sample.dagger11.sample_mvp_activity.SampleMvpActivity;
+import com.infullmobile.android.infullmvp.sample.dagger11.sample_mvp_activity.SampleMvpActivityModel;
+import com.infullmobile.android.infullmvp.sample.dagger11.sample_mvp_activity.SampleMvpActivityPresenter;
+import com.infullmobile.android.infullmvp.sample.dagger11.sample_mvp_activity.SampleMvpActivityView;
+
+import dagger.Module;
+import dagger.Binds;
+import dagger.Provides;
+
+@Module
+public abstract class SampleMvpActivityModule {
+
+    @SampleMvpActivityScope
+    @Provides
+    static SampleMvpActivityView providesMvpView() {
+        return new SampleMvpActivityView();
+    }
+
+    @SampleMvpActivityScope
+    @Provides
+    static SampleMvpActivityModel providesMvpModel() {
+        return new SampleMvpActivityModel();
+    }
+
+    @SampleMvpActivityScope
+    @Provides
+    static SampleMvpActivityPresenter providesMvpPresenter(SampleMvpActivityModel model,
```

Comment:
ugly formatting

Reply:
It's beautiful on the inside.

Seems to be related to default formatting applied when creating it via template. I'll look into that.